pennylane/pauli/pauli_arithmetic.py

```diff
@@ -192,6 +192,11 @@ def __init__(self, mapping):
                 del mapping[wire]
         super().__init__(mapping)
 
+    @property
+    def pauli_rep(self):
+        """Trivial pauli_rep"""
+        return self
```

Comment:
```suggestion
        return PauliSentence({self: 1.0})
```

Since in all other cases, `pauli_rep` is a pauli sentence.

Reply:
This was unintenional, I agree that `pauli_rep` should be a `PauliSentence`